internal/domain/indexer.go

```diff
@@ -208,10 +208,11 @@ type IndexerIRCParse struct {
 }
 
 type IndexerIRCParseLine struct {
-	Test    []string `json:"test"`
-	Pattern string   `json:"pattern"`
-	Vars    []string `json:"vars"`
-	Ignore  bool     `json:"ignore"`
+	Test         []string             `json:"test"`
+	Expectations *[]map[string]string `json:"expectations"`
```

Comment:
It might make sense to change Test from a `[]string` into a new struct that combines the case and expectations, and not sure the Expectations needs to be a pointer 🤔 

```go
type LineTest struct {
    Line   string            `json:"line"`
    Expect map[string]string `json:"expect"`
}
```

That and add a new Tests field, but keep Test to not break unofficial custom definitions.

```go
type IndexerIRCParseLine struct {
	Test    []string   `json:"test"`
        Tests   []LineTest `json:"tests"`
	Pattern string     `json:"pattern"`
	Vars    []string   `json:"vars"`
	Ignore  bool       `json:"ignore"`
}
```
Something like that maybe?